enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -34,6 +34,7 @@ import (
 	"github.com/buildbuddy-io/buildbuddy/enterprise/server/util/ext4"
 	"github.com/buildbuddy-io/buildbuddy/enterprise/server/util/vfs_server"
 	"github.com/buildbuddy-io/buildbuddy/enterprise/server/util/vsock"
+	"github.com/buildbuddy-io/buildbuddy/enterprise/vmsupport"
```

Comment:
nit: maybe import as `vmsupport_bundle` so the callsite is more clear

Reply:
Yeah, sounds good.
```suggestion
	vmsupport_bundle "github.com/buildbuddy-io/buildbuddy/enterprise/vmsupport"
```